metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1808,31 +1794,33 @@ def get_feature(self, *, feature_uri: str) -> Feature:
             description=feature_metadata['description'],
             owners=feature_metadata['owners'],
             badges=feature_metadata['badges'],
-            partition_column=feature_metadata['partition_column'],
-            owner_tags=feature_metadata['owner_tags'],
             tags=feature_metadata['tags'],
             programmatic_descriptions=feature_metadata['programmatic_descriptions'],
             last_updated_timestamp=feature_metadata['last_updated_timestamp'],
             created_timestamp=feature_metadata['created_timestamp'],
             watermarks=feature_metadata['watermarks'])
         return feature
 
-    def get_resource_generation_code(self, *, uri: str, resource_type: ResourceType) -> Query:
+    def get_resource_generation_code(self, *, uri: str, resource_type: ResourceType) -> GenerationCode:
```

Comment:
do not like how this is "fake generic". I.e. takes uri but still hardcodes `Feature_Generation_Code`.

Reply:
at first it was meant to be generic but the change in naming made it not so, I wonder if we should enforce that any generation code will follow the naming convention of {ResourceType}_Generation_Code?